.github/workflows/compute.yml

```diff
@@ -41,9 +39,14 @@ jobs:
         with:
           node-version: "20.10.0"
 
+      - run: ${{ toJSON(inputs) }}
+        shell: cat {0}
+
       - name: Generate Rewards
         uses: ./
         with:
           issueUrl: "${{ fromJSON(inputs.eventPayload).issue.html_url }}"
           evmPrivateEncrypted: "${{ secrets.EVM_PRIVATE_ENCRYPTED }}"
```

Comment:
@whilefoo I checked the current given inputs, that you can see [here](https://github.com/Meniole/conversation-rewards/actions/runs/9019007752/job/24780891776#step:4:24) and I can't seem to find it. Although, I agree that it would make sense. Might wanna have a task for this in the kernel?

Reply:
I meant to say that `evmPrivateEncrypted` should be in the `settings` field of the inputs, so basically it should be defined in the config of the plugin (in this case `conversation-rewards` should define `evmPrivateEncrypted` in the config)

Reply:
@whilefoo I changed it to be sent through the config. However the reason why these were there in the first place is to be able to generate permits, which requires both the `evmPrivateEncrypted`  and `X25519_PRIVATE_KEY` to function properly.

I guess it makes more sense that the permits are generated by the kernel and not inside the plugin itself for security reasons? But if so, I am not sure about the communication workflow because it looks like a lot of back and forth.

Reply:
Yeah it'd probably make sense that they are generated by the kernel, maybe through a standard field for example `permits` in the plugin output but that would still mean that there needs to be another plugin after to make the comment...so there's not a clean and simple way to generate them by the kernel. I guess for now we can let them be generated by conversation-rewards plugin

Reply:
Okay, let's do that then. The `EVM_PRIVATE_ENCRYPTED` defined in the config matches the `X25519_PRIVATE_KEY` which is overridden for this repository, to use an empty wallet for testing.